test/downstream/observables.jl

```diff
@@ -53,3 +56,25 @@ plot(sol,vars=(lorenz2.x,α))
 plot(sol,vars=α)
 plot(sol,vars=(t,α))
 =#
+
+@variables t x(t)
+@parameters τ
+D = Differential(t)
+@variables RHS(t)
+@named fol_separate = ODESystem([ RHS  ~ (1 - x)/τ,
+                                  D(x) ~ RHS ])
+fol_simplified = structural_simplify(fol_separate)                                  
+
+prob = ODEProblem(fol_simplified, [x => 0.0], (0.0,10.0), [τ => 3.0])
+sol = solve(prob, Tsit5())
+
+@test sol[x] isa Vector
+@test sol[x,2] isa Float64
+@test sol[x,:] isa Vector
+@test length(sol[x,1:5]) == 5
+@test sol[RHS] isa Vector
+@test sol[RHS,3] isa Float64
+@test length(sol[RHS,5:10]) == 6
+
+interpolated_sol = sol(0.0:1.0:10.0)
+@test interpolated_sol[RHS] ≈ (1 .- interpolated_sol[x])./3.0
```

Comment:
The previous Lorenz system-based example found in `master` fails for some reason outside of tests in `structural_simplify` step. I have created an issue #68 .

For now, I have replaced it with a simpler example that can be used for all old and new checks.

Reply:
https://github.com/SciML/SciMLBase.jl/issues/68